quickstart/routes/web.php

```diff
@@ -16,3 +19,6 @@
 Route::get('/', function () {
     return view('welcome');
 });
+Route::resource('category', CategoryController::class);
```

Comment:
```suggestion
Route::resource('categories', CategoryController::class);
```
số nhiều, sửa hết đi em